### PR TITLE
Adds Limitless Prediction Market Blog

### PR DESCRIPTION
**Pull request by SvenMuller95**

### CODE REVIEW

**Review by Jordy-Baby — APPROVED** [review]

File: blog/2024-09-27-case-study-limitless.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+<!--truncate-->
+
+Prediction markets are emerging as a new craze in decentralized finance, gaining significant attention and establishing themselves as major players in this cycle's narrative. The upcoming US election is driving this interest, raising questions about the sustainability of trading volume after the election. Others strongly believe that prediction markets have found product-market fit, capturing mainstream attention and joining other established categories like stablecoins, NFTs, meme coins, and lending and borrowing protocols.
+
+This new paradigm of prediction markets leverages blockchain technology and cryptocurrency as infrastructure, offering better user experiences than what existed before. Low fees, global accessibility, permissionless operation, and ease of coordination contribute to their adoption and success. Built on a public blockchain, these platforms ensure that all transactions are transparent, censorship-resistant, and executed quickly and accurately by smart contracts. This also makes them accessible to anyone — unlike traditional prediction market services, which are often restricted by financial regulations.
```
Comment (moose-code) [inline blog/2024-09-27-case-study-limitless.md:13]: Last sentence here isn't good! This shouldn't be what blockchain is about, getting around regulations. Its about the transparency and fairness. Lets remove this last sentence

File: blog/2024-09-27-case-study-limitless.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+Overview of Prediction Markets
+
+<img src="/blog-assets/case-study-limitless-3.png" alt="Cover Image Limitless Prediction Markets Case Study" width="100%"/>
+
+Currently, Polymarket is the world's largest prediction market. Deployed on Polygon, Polymarket has experienced exponential growth, with elections accounting for 85% of its total volume. In July 2024, the platform generated an impressive $137.3 million in weekly trading volume. Data also suggests that non-election prediction markets have been increasing, such as betting on crypto prices, sports results, or social events.
+
+Azuro Protocol stands out for its sports betting markets, which provide the tooling, oracle, and liquidity solution for EVM chains to host sports-specific prediction markets. B.E.T built on top of the Drift Protocol, and Hedgehog, are two popular prediction markets on Solana, with B.E.T. recently reaching $20 million daily volume.
+
+Limitless Labs, a new daily prediction market on Base focusing on price action and sporting events, recently hit [$2 million in volume](https://x.com/trylimitless/status/1833621283290157216). A week prior, Limitless was doing $100k in volume, suggesting its traction.
```
Comment (moose-code) [inline blog/2024-09-27-case-study-limitless.md:40]: suggesting its traction ... is growing?

**Review by moose-code — CHANGES_REQUESTED** [review]
Great! A couple comments I left to change. You might also want to tag limitless to review this?

**Review by moose-code — APPROVED** [review]
LGTM

**Review by Jordy-Baby — APPROVED** [review]

**Pull request merged**